From: Steve Langasek <steve.langasek@ubuntu.com>
Date: Wed, 25 May 2022 18:22:01 +0800
Subject: Drop etc/adjtime from scripts/build/lb_chroot_hacks.

/etc/adjtime is not  needed and should not be used in Ubuntu.
Forwarded: no
Last-Update: 2013-09-27
---
 scripts/build/lb_chroot_hacks | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/scripts/build/lb_chroot_hacks b/scripts/build/lb_chroot_hacks
index d79ccd5..9d32be2 100755
--- a/scripts/build/lb_chroot_hacks
+++ b/scripts/build/lb_chroot_hacks
@@ -218,9 +218,6 @@ esac
 # people might do in local hooks, and there might be no initrds at all.
 find chroot/boot -name 'initrd*' -print0 | xargs -r -0 chmod go+r
 
-# Remove build systems clock drift
-echo "0.0 0 0.0" > chroot/etc/adjtime
-
 # Remove cruft
 rm -f chroot/boot/initrd*bak*
 rm -f chroot/etc/apt/trusted.gpg~
@@ -229,6 +226,7 @@ rm -f chroot/etc/gshadow- chroot/etc/shadow-
 rm -f chroot/var/cache/debconf/*-old
 rm -f chroot/var/lib/dpkg/*-old
 rm -f chroot/var/log/apt/term.log
+rm -f chroot/etc/adjtime
 
 # Delete and re-create temporary directories
 TEMP_DIRS="/tmp /var/tmp"
@@ -277,7 +275,7 @@ then
 	done
 
 	# Config files which need to be RW
-	COW_FILES="/etc/adjtime /etc/fstab /etc/hostname /etc/hosts /etc/live.conf /etc/network/interfaces /etc/resolv.conf /etc/udev/rules.d/*persistent-net.rules /etc/udev/rules.d/*persistent-cd.rules /etc/X11/xorg.conf"
+	COW_FILES="/etc/fstab /etc/hostname /etc/hosts /etc/live.conf /etc/network/interfaces /etc/resolv.conf /etc/udev/rules.d/*persistent-net.rules /etc/udev/rules.d/*persistent-cd.rules /etc/X11/xorg.conf"
 
 	# Where we will store RW config files
 	RW_DIRECTORY="/var/lib/live"
